lib/formats.js

```diff
@@ -2,11 +2,25 @@
 
 var Formats = module.exports = {};
 
-Formats.swagger_1 = require('./formats/swagger_1.js');
-Formats.swagger_2 = require('./formats/swagger_2.js');
-Formats.openapi_3 = require('./formats/openapi_3.js');
-Formats.api_blueprint = require('./formats/api_blueprint.js');
+// some modules do require optionalDependencies, we silently catch loading if
+// that does not work, and we will warn user when he will try using it
+try {
+  Formats.swagger_1 = require('./formats/swagger_1.js');
+} catch (e) {}
```

Comment:
Can you check to make sure `e` is a `MODULE_NOT_FOUND` error, and `throw e` otherwise? Otherwise syntax errors will throw a confusing message about optional dependencies.

Reply:
right... PR updated.